tests/css_case/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -364,13 +364,13 @@ $KeepScssVar: val;
 
   @KeepDetachedRuleset();
 
-  &: EXTEND(.Keep ALL);
+  &:EXTEND(.Keep ALL);
```

Comment:
Anyone know what is this? space matters?

Reply:
I think I added this test case when I added support for lowercasing of stuff back in the day. Testing on http://lesscss.org/less-preview/, uppercase `EXTEND` does not appear to be valid Less. So it’s a weird test case. `&:extend(.Keep ALL)` seems to be valid syntax, while `&: extend(.Keep ALL)` is a syntax error.

Reply:
what output is expect?

Reply:
My guess is that the input should be changed to `&:extend(.Keep ALL)` and the output should be the same (`&:extend(.Keep ALL)`).